src/components/sbb-alert/sbb-alert.tsx

```diff
@@ -185,69 +180,75 @@ export class SbbAlert implements ComponentInterface, LinkProperties {
     };
   }
 
-  public render(): JSX.Element {
-    return (
+  protected override render(): TemplateResult {
+    return html`
       <div
         class="sbb-alert__transition-wrapper"
-        ref={(el): void => {
-          this._transitionWrapperElement = el;
-        }}
+        ${ref((el): void => {
+          this._transitionWrapperElement = el as HTMLElement;
+        })}
       >
         <div
           class="sbb-alert"
-          ref={(el): void => {
+          ${ref((el): void => {
             const isFirstInitialization = !this._alertElement;
 
-            this._alertElement = el;
+            this._alertElement = el as HTMLElement;
             if (isFirstInitialization) {
               this._initFadeInTransitionStyles();
             }
-          }}
+          })}
         >
           <span class="sbb-alert__icon">
-            <slot name="icon">{<sbb-icon name={this.iconName || 'info'} />}</slot>
+            <slot name="icon">
+              <sbb-icon name=${this.iconName || 'info'}></sbb-icon>
+            </slot>
           </span>
           <span class="sbb-alert__content">
             <sbb-title
               class="sbb-alert__title"
-              level={this.titleLevel}
-              visual-level={this.size === 'l' ? '3' : '5'}
+              level=${this.titleLevel}
+              visual-level=${this.size === 'l' ? '3' : '5'}
               negative
             >
-              <slot name="title">{this.titleContent}</slot>
+              <slot name="title">${this.titleContent}</slot>
             </sbb-title>
             <p class="sbb-alert__content-slot">
-              <slot />
+              <slot></slot>
             </p>
-            {this.href && (
-              <Fragment>
-                <span aria-hidden="true">&nbsp;</span>
-                <sbb-link {...this._linkProperties()} variant="inline" negative>
-                  {this.linkContent ? this.linkContent : i18nFindOutMore[this._currentLanguage]}
-                </sbb-link>
-              </Fragment>
-            )}
+            ${this.href
+              ? html` <sbb-link ${spread(this._linkProperties())} variant="inline" negative>
+                  ${this.linkContent ? this.linkContent : i18nFindOutMore[this._currentLanguage]}
+                </sbb-link>`
+              : nothing}
```

Comment:
Did you mean to delete the `<span aria-hidden="true">&nbsp;</span>`?

Reply:
Yep, I noticed that with Lit we have a different handling of spaces between inline elements and this was not needed anymore.